recipes/libsafec/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+import glob
+import os
+
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibSafeCConan(ConanFile):
+    name = "libsafec"
+    description = "This library implements the secure C11 Annex K[^5] functions" \
+            " on top of most libc implementations, which are missing from them."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rurban/safeclib"
+    license = "MIT"
+    topics = ("conan", "safec", "libc")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    build_requires = "autoconf/2.69", "libtool/2.4.6"
+    exports_sources = "patches/*"
+
+    __autotools = None
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    @property
+    def _supported_compiler(self):
+        compiler = self.settings.compiler
+        version = tools.Version(self.settings.compiler.version)
+        if compiler == "Visual Studio":
+            return False
+        if compiler == "gcc" and version < "5":
+            return False
+        return True
+
+    def configure(self):
+        if not self._supported_compiler:
+            raise ConanInvalidConfiguration(
+                "libsafec doesn't support compiler: {} on OS: {}.".format(
+                    self.settings.compiler, self.settings.os))
```

Comment:
So the error mention makes it look like GCC can work
```suggestion
                "libsafec doesn't support {}/{}".format(
                    self.settings.compiler, self.settings.compiler.version))
```